lib/compress/zstd_compress_internal.h

```diff
@@ -53,11 +53,15 @@ typedef struct ZSTD_prefixDict_s {
 } ZSTD_prefixDict;
 
 typedef struct {
+    /* Huffman data
+     * Must be before the FSE data.
```

Comment:
Indicate why (I guess it's because you want to `memcpy()` FSE tables)

Reply:
Yeah, I kept on forgetting to update the copy when `nbSeq == 0`, so I wanted to write it in a way where it gets automatically fixed.

The good news is that even if it is broken, which should be hard because of the static asserts, I know the regression tests will find it.